Adyen/UserInterface/01a_Connections.md

```diff
@@ -0,0 +1,108 @@
+[!! Manage Adyen connection](../Integration/02_ManageAdyenConnection.md)
+
+
+# CONNECTIONS (Adyen)
+
+The CONNECTIONS tab provides an overview of all connections created for *Payments*. It also allows you to configure these connections.   
+For detailed information on general UI functions of the *CONNECTIONS* tab, see [Connections](../../Payments/UserInterface/08a_Connections.md) in the *Payments* documentation.
+
+
+## Edit Adyen connection &ndash; Credentials
+
+*Payments > Settings > Tab CONNECTIONS > Select Adyen&trade; connection > Tab Credentials*
+
+![Edit Adyen credentials](../../Assets/Screenshots/Adyen/Integration/EditCredentials.png "[Edit Adyen credentials]")
+
+- *Driver*  
+    Selected driver name. This drop-down list is read-only.
+
+**Credentials**
+
+The *Credentials* tab displays the connection credentials that you defined in Adyen&trade;. They are a copy of these credentials. 
+
+- *Checkout API endpoint*   
+   Endpoint of the checkout API that is managed in Adyen&trade;. For detailed information on the checkout API, see [Adyen Checkout API](https://docs.adyen.com/api-explorer/Checkout/71/overview "[https://docs.adyen.com/api-explorer/Checkout/71/overview]") in the Adyen&trade; documentation. For the testing environment it is `https://checkout-test.adyen.com/checkout/`, for the live account you get the endpoint directly from Adyen&trade;. 
+     
+- *Checkout API key*    
+   Key of the *Checkout API* endpoint that is managed in the Adyen&trade; backend.
+
+- *Payment API endpoint*   
+   Endpoint of the *Payment API*. For the test environment, it is `https://pal-test.adyen.com/pal/servlet/Payment/`, for the live account you get the endpoint directly from Adyen&trade;.
+
+- *Payment user*   
+   Username of the technical user that is managed in the Adyen&trade; backend.
+
+- *Payment password*   
+   Password of the *Checkout* API that is managed in the Adyen&trade; backend.
+
+ - *Report user*  
+     Username of the *Report API* that is managed in the Adyen&trade; backend.
+
+ - *Report password*    
+    Password of the *Report API* you have created in the settings of the Adyen&trade; backend.
+
+- *Notification user*  
+    Edit the username of the technical user. You can edit the notification user both in Actindo or in Adyen&trade;. If you do any changes here, remember to follow these changes in the Adyen&trade; webhook configuration. For detailed information, see [Create Adyen webhook](../Integration/01_ManageAdyenSettings.md#create-adyen-webhook).
```

Comment:
Wenn ich mich nicht irre, "both" kann/soll man nur mit "and" verwenden, also "both in Actindo and in Adyen".